src/reformat.c

```diff
@@ -204,6 +204,16 @@ avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb)
         return AVIF_RESULT_NOT_IMPLEMENTED;
     }
 
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+    if (state.mode == AVIF_REFORMAT_MODE_YCGCO_RE || state.mode == AVIF_REFORMAT_MODE_YCGCO_RO) {
+        const int bitOffset = (state.mode == AVIF_REFORMAT_MODE_YCGCO_RE) ? 2 : 1;
+        const int maxValue = (1 << (state.yuvDepth - bitOffset)) - 1;
```

Comment:
Nit/optional: Here we can compare `state.yuvDepth` or `image->depth` with `rgb->depth` directly and skip the calculation of `maxValue`, e.g.,

```
        if (state.yuvDepth - bitOffset != rgb->depth) {
            return AVIF_RESULT_UNSUPPORTED_DEPTH;
        }
```

Reply:
Done